tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.settings.app import Settings
+from marionette_driver import By
+
+
+class TestSettingsRTL(GaiaImageCompareTestCase):
+    _page_locator = (By.ID, 'root')
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+
+    def test_settings_app(self):
+
+        settings = Settings(self.marionette)
+        settings.launch()
+
+        self.take_screenshot('main')
+
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
```

Comment:
hmm, for me, the total_height = comes out as 539.33, while the screen_height comes as 569.  The page locator in Settings is (By.ID, 'root')  I don't think it draws the entire menu list and compute its height - i think it draws as you scroll, so it would be difficult to get the total height.